app/scripts/views/sign_up.js

```diff
@@ -177,6 +177,14 @@ define(function (require, exports, module) {
         shouldFocusPassword: autofocusEl === 'password',
         showSyncSuggestion: this.isSyncSuggestionEnabled()
       };
+      if (this.getUserAgent().hasWebChannelSupport()) {
```

Comment:
@shane-tomlinson @vershwal can we use `fx_desktop_v3` or do we need to change it depending on Desktop vs Mobile